Library/SharedFunctions.swift

```diff
@@ -106,7 +106,16 @@ internal func minAndMaxPledgeAmount(forProject project: Project, reward: Reward?
   case .none, .some(Reward.noReward):
     return (Double(country.minPledge ?? 1), Double(country.maxPledge ?? 10_000))
   case let .some(reward):
-    return (reward.minimum, Double(country.maxPledge ?? 10_000))
+    let min: Double
+
+    /// Account for the case where the originally selected reward pricing, for this backing, has since changed to late pledge pricing. We should always use the original pricing at the time of the backing over the most current state.
+    if project.personalization.backing?.isLatePledge == true {
```

Comment:
Yeah currently, if there's no backing, this would default to the `pledgeAmount`, unless there isn't one, then it would default to the `minimum`.

I'll add a guard statement so that if there isn't a backing we default to the original minimum logic!

```
guard let backing = project.personalization.backing else {
      return (reward.minimum, Double(country.maxPledge ?? 10_000))
    }
```